content/quickstart.md

```diff
@@ -98,5 +105,3 @@ $ watch oc get pods
 ```
 
 You should now be able to open `https://syndesis.$(minishift ip).xip.io` in your browser.
```

Comment:
Any idea when a switch to `.nip` is possible ? Wonder whether there is already an issue open at minishift wrt to the problematic certs ? (still not really clear what the issue is and whether it part of `oc cluster up` or `minishift)